src/builtin_proto.h

```diff
@@ -12,11 +12,11 @@ extern "C" {
 #ifdef DEFINE_BUILTIN_GLOBALS
 #define DECLARE_BUILTIN(name) \
     JL_CALLABLE(jl_f_##name); \
-    jl_value_t *jl_builtin_##name
+    jl_value_t *jl_builtin_##name JL_GLOBALLY_ROOTED
 #else
 #define DECLARE_BUILTIN(name) \
     JL_CALLABLE(jl_f_##name); \
-    extern jl_value_t *jl_builtin_##name
+    extern jl_value_t *jl_builtin_##name JL_GLOBALLY_ROOTED
```

Comment:
This is a bit of a long shot for working around analyzegc https://build.julialang.org/#/builders/65/builds/1574/steps/4/logs/stdio

But I wonder if there is a reason why this is not used?  Maybe I'm doing something stupid here?

If this is not the right way to do it, maybe I need to do something like this?

```diff
diff --git a/src/builtins.c b/src/builtins.c
index 440b69629e..c2576d553d 100644
--- a/src/builtins.c
+++ b/src/builtins.c
@@ -369,6 +369,7 @@ JL_CALLABLE(jl_f_is)
         jl_function_t *fix2 = jl_get_function(jl_base_module, "Fix2");
         if (fix2 == NULL)
             jl_undefined_var_error(jl_symbol("Fix2"));
+        JL_GC_PROMISE_ROOTED(jl_builtin_is);
         return jl_call2(fix2, jl_builtin_is, args[0]);
     }
     if (args[0] == args[1])
@@ -440,6 +441,7 @@ JL_CALLABLE(jl_f_isa)
         jl_function_t *fix2 = jl_get_function(jl_base_module, "Fix2");
         if (fix2 == NULL)
             jl_undefined_var_error(jl_symbol("Fix2"));
+        JL_GC_PROMISE_ROOTED(jl_builtin_isa);
         return jl_call2(fix2, jl_builtin_isa, args[0]);
     }
     JL_TYPECHK(isa, type, args[1]);
```